specification/sql/resource-manager/readme.md

```diff
@@ -417,6 +417,8 @@ input-file:
 - ./Microsoft.Sql/preview/2021-11-01-preview/DatabaseAdvancedThreatProtectionSettings.json
 - ./Microsoft.Sql/preview/2021-11-01-preview/ServerAdvancedThreatProtectionSettings.json
 - ./Microsoft.Sql/preview/2021-11-01-preview/ManagedServerDnsAliases.json
+- ./Microsoft.Sql/preview/2022-02-01-preview/DatabaseUsages.json
```

Comment:
add to bottom or sort as alphabetical order?

Reply:
done in alphabetical order

Reply:
I corrected the order and the name as well (Databases and ReplicationLinks)